nextflow.config

```diff
@@ -96,54 +123,76 @@ profiles {
         podman.enabled         = false
         shifter.enabled        = false
         charliecloud.enabled   = false
+        apptainer.enabled      = false
     }
     docker {
         docker.enabled         = true
         docker.userEmulation   = true
+        conda.enabled          = false
         singularity.enabled    = false
         podman.enabled         = false
         shifter.enabled        = false
         charliecloud.enabled   = false
+        apptainer.enabled      = false
     }
     arm {
         docker.runOptions = '-u $(id -u):$(id -g) --platform=linux/amd64'
     }
     singularity {
         singularity.enabled    = true
         singularity.autoMounts = true
+        conda.enabled          = false
         docker.enabled         = false
         podman.enabled         = false
         shifter.enabled        = false
         charliecloud.enabled   = false
+        apptainer.enabled      = false
     }
     podman {
         podman.enabled         = true
+        conda.enabled          = false
         docker.enabled         = false
         singularity.enabled    = false
         shifter.enabled        = false
         charliecloud.enabled   = false
+        apptainer.enabled      = false
     }
     shifter {
         shifter.enabled        = true
+        conda.enabled          = false
         docker.enabled         = false
         singularity.enabled    = false
         podman.enabled         = false
         charliecloud.enabled   = false
+        apptainer.enabled      = false
     }
     charliecloud {
         charliecloud.enabled   = true
+        conda.enabled          = false
+        docker.enabled         = false
+        singularity.enabled    = false
+        podman.enabled         = false
+        shifter.enabled        = false
+        apptainer.enabled      = false
+    }
+    apptainer {
+        apptainer.enabled      = true
+        conda.enabled          = false
         docker.enabled         = false
         singularity.enabled    = false
         podman.enabled         = false
         shifter.enabled        = false
+        charliecloud.enabled   = false
     }
     gitpod {
         executor.name          = 'local'
         executor.cpus          = 16
         executor.memory        = 60.GB
     }
-    test      { includeConfig 'conf/test.config'      }
```

Comment:
Will it better to keep test and rename any of test_targeted, test_umis, or test_screening to test?